### PR TITLE
Set LAPACK path

### PR DESCRIPTION
**Pull request by leouieda**
Trying to avoid linking problems with OSX Accelerate framework.
Suggested by @isuruf in #3

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ requirements:
     - blas 1.1 {{ variant }}
     - curl
     - pcre
+    - lapack
```
Comment (isuruf) [inline recipe/meta.yaml]: Replace this with `openblas 0.2.19|0.2.19.*`. `openblas` is an optimized blas and lapack library and is better than `lapack` which is the reference implementation with no optimization

Comment (leouieda) [reply recipe/meta.yaml]: But should I keep `blas 1.1`? 

Are you sure `openblas` has LAPACK? I can't find any mention of it on their website.

Comment (isuruf) [reply recipe/meta.yaml]: `blas 1.1` is there to have one implementation of blas per env. @ocefpaf, @jakirkham can explain better than me.

Yes, I'm sure `openblas` has lapack

Comment (jakirkham) [reply recipe/meta.yaml]: Here are the [docs]( https://conda-forge.org/docs/blas.html ). Yes `openblas` implements both the BLAS and LAPACK interfaces within the same library. So it serves both purposes.

Comment (leouieda) [reply recipe/meta.yaml]: OK, thanks for the explanation

Comment (jakirkham) [reply recipe/meta.yaml]: Sorry I realize that doc mention was super terse. If you have any questions, please feel free to ask.

Comment (leouieda) [reply recipe/meta.yaml]: @jakirkham not at al. I didn't think to look in the conda-forge docs and it's good to know that the information is there. Reply time for all conda-forge is impressively low!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ source:
   sha256: 288509d28ed062ddf8b08212f65afb3aa07762a9a84bc5da9f7132016c8dd31f
 
 build:
-  number: 2
+  number: 3
```
Comment (jakirkham) [inline recipe/meta.yaml]: FYI with `openblas` this should start at `200` and go up from there or be reset to `200` when making a new release.

Comment (leouieda) [reply recipe/meta.yaml]: @jakirkham OK. So should I maintain multiple builds/branches with and without openblas?

Comment (leouieda) [reply recipe/meta.yaml]: @jakirkham should I update the build number now or wait until the next release?

**Pull request merged**